templates/telecommunications/index.html

```diff
@@ -0,0 +1,208 @@
+    </div>
+    <div class="col-4 p-divider__block">
+      <h2 class="p-heading--three">Expert knowledge and experience</h2>
+      <p>Our team of specialists operates production CSP environments around the globe, covering the entire environment life-cycle, from deployment to upgrades and onboarding virtual network functions &mdash; enabling you to realise the true potential of your cloud.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>Why choose Canonical?</h2>
+    <ul class="p-list is-split">
+      <li class="p-list__item is-ticked">Keep <abbr title="Capital expenditure">CAPEX</abbr> and <abbr title="Operating expenses">OPEX</abbr> costs down with virtualisation and automation</li>
+      <li class="p-list__item is-ticked">Peace of mind with a pre-tested solution</li>
+      <li class="p-list__item is-ticked">Get started quicker with our speedy onboarding process</li>
+      <li class="p-list__item is-ticked">High-availability <abbr title="ervice-level agreement">SLA</abbr>s aligned with accepted industry practices</li>
+      <li class="p-list__item is-ticked">Stay ahead with easy upgrades between OpenStack versions and fast access to the latest OpenStack features with 100% upstream integration</li>
+    </ul>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>The complete cloud automation toolset</h2>
+      <p>Operational discipline &mdash; built on automation and repeatability &mdash; has long been crucial to the profitability of CSPs. Canonical applies the same principles to the way clouds are built and run.  From design to deployment and operations, every aspect is managed with Canonical&rsquo;s proven automation toolset, including provisioning tool MAAS, management tool Landscape, and application modeling tool Juju.</p>
+      <ul class="p-list">
+        <li><a href="/cloud/juju">Find out more about Juju&nbsp;&rsaquo;</a></li>
+        <li><a href="/server/maas">Find out more about MAAS&nbsp;&rsaquo;</a></li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>NFVI cloud build</h2>
+      <p>Canonical&rsquo;s Foundation Cloud Build for Telcos is a pre-tested, pre-integrated OpenStack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about NFVI cloud build</span>&nbsp;&rsaquo;</a></p>
```

Comment:
I know this matches the copy doc but there is no `NFVI` on the foundation-cloud. I'll look into it.